### PR TITLE
Add GameInfo feature

### PR DESCRIPTION
**Pull request by fikusmaksymilian**

### CODE REVIEW

**Review by kejmil4 — APPROVED** [review]
ok

File: FMSim/FMInterface/Info/GameInfo.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class GameInfo
+{
+    public static void DisplayGameInfo()
+    {
+        List<string> infoList = new List<string>
+        {
+            "FMSim",
+            "Copyright © 2024",
+            "Team:",
+            "Project Manager: Michal Krol",
```
Comment (MishutkAviation) [inline FMSim/FMInterface/Info/GameInfo.cs]: łśćżąę

**Review by MishutkAviation — CHANGES_REQUESTED** [review]

**Review by JanGrzeskowiak — CHANGES_REQUESTED** [review]
Wrong Position for me my friend, please fix

**Review by MishutkAviation — APPROVED** [review]

**Review by JanGrzeskowiak — APPROVED** [review]
Kys

**Pull request merged**